protocol/messenger.go

```diff
@@ -4027,10 +4029,13 @@ func (m *Messenger) MessageByChatID(chatID, cursor string, limit int) ([]*common
 		}
 
 	}
-	if m.httpServer != nil {
-		for idx := range msgs {
-			m.prepareMessage(msgs[idx], m.httpServer)
-		}
+
+	if m.httpServer == nil {
```

Comment:
i'm not sure, but do we always have `httpServer`?

Reply:
@MishkaRogachev : this file was reverted.